cmd/dtab.go

```diff
@@ -93,7 +107,7 @@ var (
 				if err != nil {
 					return err
 				}
-				_, err = ctl.Create(name, dtabstr)
+				_, err = ctl.Create(name, dtabstr, dtabJson)
```

Comment:
Hmm, not sure what I am doing wrong locally, but I couldn't get `create --json` to work.

```
$ cat test.dtab 
{"dtab": [{"prefix":"/http","dst":"/$/inet/veggies/7003"}]}
$ go run main.go dtab --base-url http://192.168.99.100:4180 --json create newns test.dtab 
Error: unexpected response: 400 Bad Request
```